core/templates/dev/head/pages/exploration_player/AudioTranslationManagerService.js

```diff
@@ -65,6 +66,15 @@ oppia.factory('AudioTranslationManagerService', [
           _allAudioLanguageCodesInExploration.length >= 1) {
         _currentAudioLanguageCode = _allAudioLanguageCodesInExploration[0];
       }
+
+      if (_currentAudioLanguageCode === null &&
+          _allAudioLanguageCodesInExploration.length == 0) {
+        _currentAudioLanguageCode = 
+          _explorationLanguageCode +
```

Comment:
It's not a given that the exp language code has an associated autogeneratable translation, is it?

Reply:
Right, fixed.